.codecov.yml

```diff
@@ -13,8 +13,9 @@ coverage:
           - unit
           - integration
         paths:
-          - src
+          - backend
+          - lib
         only_pulls: true
 
 ignore:
-  - '*.js'
+  - "*.js"
```

Comment:
Why are we ignoring coverage for the javascript files?

Reply:
because it a typescript repo